modules/bigtable-instance/main.tf

```diff
@@ -54,17 +51,17 @@ resource "google_bigtable_instance_iam_binding" "default" {
 }
 
 resource "google_bigtable_table" "default" {
-  for_each      = local.tables
+  for_each      = var.tables
```

Comment:
this will fail if the user passes a `null` value to the variable, I would add `nullable = false` to the variable definition, and set a default value of `{}`

Reply:
New commit with those changes added. Thanks for the review, I did not test that case...